backend/grant/rfp/views.py

```diff
@@ -9,20 +8,18 @@
 
 
 @blueprint.route("/", methods=["GET"])
-@endpoint.api()
 def get_rfps():
     rfps = RFP.query \
         .filter(or_(
-            RFP.status == RFPStatus.LIVE,
-            RFP.status == RFPStatus.CLOSED,
-        )) \
+        RFP.status == RFPStatus.LIVE,
+        RFP.status == RFPStatus.CLOSED,
+    )) \
```

Comment:
👀

Reply:
Eek, formatting wars continue. I don't feel it's worth digging into at this point, but we'll get it fixed up eventually